spec/models/builder_spec.rb

```diff
@@ -173,37 +184,16 @@ def file_contains(path, fragment)
       #   'spec.add_dependency "rails-assets-desandro--get-size", ">= 1.1.4", "< 2.0"'
     end
 
-    # This is strange example because colorbox has only
```

Comment:
It makes no sense to "select first example colorbox.css". This is obviously a bug; it's not in the `main` of the bower JSON file.